src/app/docs/description-list/description-list-docs.component.html

```diff
@@ -0,0 +1,90 @@
+<sky-docs-demo-page
+  moduleName="SkyDescriptionListModule"
+  moduleSourceCodePath="src/app/public/modules/description-list/"
+  pageTitle="Description list"
+>
+  <sky-docs-demo-page-summary>
+    Description lists display term-desription pairs.
```

Comment:
```suggestion
    Description lists display lists of terms and pairs each term with a description.
```
I know this parallels the intro in definition list, but I think it will be clearer if we flesh this out just a tiny bit to use friendlier language. And we need to fix the misspelling in the second instance of `description` anyway.